src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -193,7 +193,7 @@ export default function ToeristischeVerhuur() {
                   </>
                 )}
               </Heading>
-              {getYear(new Date()) === 2021 && (
+              {getYear(new Date()) === 2021 ? (
                 <p className={styles.DetailText}>
                   Het aantal resterende nachten is gebaseerd op meldingen voor
                   ingeplande en afgelopen verhuur die u <b> vanaf 19 juli</b>{' '}
```

Comment:
Waarom heb je `<b/>` gebruikt? https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em